### ISSUE TITLE
Editor Slide Animation UX?

### ISSUE THREAD

**Issue by itsdavidmorgan** [Leeflets/leeflets#71]
I like the sliding animation of the editor, but I think it becomes a bit repetitive when switching between sections and settings — constantly closing and re-opening.

What do you guys think about the window sliding open initially, then remaining open when switching between sections until the editor window is manually closed? Possibly a subtle fade animation of the content between sections after the window is open?

**Comment by jptksc** [comment]
I've never really thought about it, so it must not bother me. Either way, I suppose what you're saying here makes a bit more sense in theory. No harm in testing it out to see what kind of a difference it makes.

**Comment by bradt** [comment]
As I was coding it up, I was thinking that it would be nicer if as one panel was closing, the other would slide from underneath and you'd start to see it about halfway. Would make it quite a bit faster and more responsive. What do you think of that?

I didn't end up coding it at the time because it was more than 5 min thing, so I decided not to make it a priority. In fact, I'm still not sure which milestone this should go under?

Fading in new content may not work so well because not all the panels are the same width. I think it would be awkward to have the panel content fade in and the panel itself grow/shrink in size. At the very least, it wouldn't be as slick as the sliding panels.

**Comment by jptksc** [comment]
I really like your idea of sliding panels simultaneously. Seems like a good compromise.

**Comment by itsdavidmorgan** [comment]
Yeah sliding panels simultaneously sounds like a good solution. I don't think it's a high priority. 

It's one of those things I'd have to see in action to know if it was any better than the current flow. I'd save it for after the beta. We could get more feedback from the public then as well.

**Comment by bradt** [comment]
Cool, added to the Launch milestone.
